doc/whats_new.rst

```diff
@@ -25,6 +25,9 @@ New features
    - Added the :class:`ensemble.IsolationForest` class for anomaly detection based on
      random forests. By `Nicolas Goix`_.
 
+   - Added new supervised learning algorithm: :ref: `Multi-layer Perceptron <multi-layer-perceptron>`
```

Comment:
linking doesn't have anything to do with dev or not dev. The link is internal to this build of the website.